docs/guides/documentation/releases/README.md

```diff
@@ -0,0 +1,94 @@
+# Releases
+
+<pre align="center">Streamline software releases through consistent templates, automated aggregation and notifications.</pre>
+
+## Introduction
+
+**Background**: Making releases can be a chore. From ensuring the release notes have helpful information to aggregating many updates and notifying your users - doing the process right can be time consuming and fraught with misunderstandings. In this guide we recommend a standardized, automated way to notify your users of new releases. For example, we recommend using a `.github/release.yml` template to allow for more readable release notes, showcasing the importance of each update. We also provide automation for aggregating releases from many repositories into a super-release that provides a holistic view of project progress. Finally, we recommend automated notifications through GitHub integrations to help keep everyone informed.
+
+**Use Cases**:
+- Automating release notes generation with a readable template
+- Aggregating multiple repository releases for projects with many repositories
+- Streamlining release notifications to keep teams and users informed automatically
+
+---
+
+## Prerequisites
+* A GitHub account and repository
+* Basic understanding of GitHub Actions and workflows
+* Access to IM channels like Slack for integration
+
+---
+
+## Quick Start
+**[⬇️ Recommended GitHub release.yml Template](release.yml)**
+
+_A customizable GitHub-specific release template. Customize it to fit your project's labeling scheme and presentation preferences for release notes and place it in your GitHub repository's `.github/release.yml` path._
+
+**[⬇️ Python Script to Aggregate GitHub Releases](gh_aggregate_release_notes.py)**
+
+_A Python script that aggregates release notes from many repositories and generates a "super" release text._
+
+NOTE: You'll need a configuration file to use the above Python script. See step 2.2 below for an example file that you can customize for your project.
+
+
+---
+
+## Step-by-Step Guide
+
+1. **Customize Release Notes**:
+   - Create a `.github/release.yml` file in your repository.
+   - Download our [GitHub release notes template](release.yml) and place the contents into your `.github/release.yml` file.
+   - Customize our recommended template as a baseline and to configure how release notes are generated based on your project's issue labels.
+   
+2. **Aggregate Releases**:
+   We recommend using a script to automatically aggregate release notes from multiple repositories. Details below.
+
+   2.1 Download our script 
+   - Copy/download our script to your local machine
+      ```
+      curl -o gh_aggregate_release_notes.py https://raw.githubusercontent.com/NASA-AMMOS/slim/issue-97/docs/guides/documentation/releases/gh_aggregate_release_notes.py
```

Comment:
Personal taste: in documentation like this, I always prefer the `--long` format command-line arguments as they feel more "documentary" 😁 , so `-o` → `--output`

Reply:
Thanks @nutjob4life - fixed in latest commit. 

@jpl-jengelke - will keep this comment unresolved so I don't forget!